srcpkgs/srb2/files/srb2.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Sonic Robo Blast 2
+Encoding=UTF-8
+Exec=srb2
+Icon=srb2
+Terminal=false
+Type=Application
+StartupNotify=false
+Categories=Application;Game;
```

Comment:
I don't like vendoring in desktop files... Since this is an open source project, could you try to PR them this as an addition?